internal/ntt/manifest.go

```diff
@@ -315,32 +315,81 @@ func (suite *Suite) TestHook() (*fs.File, error) {
 	return nil, nil
 }
 
+// ParametersDir return the absolute path retrieven from either
+// * parameters_dir field from manifest
+// * suites root path.
+func (suite *Suite) ParametersDir() (string, error) {
```

Comment:
We have a contract that all manifest keys may be overwritten by environment variables.
You should also add a check for `suite.Getenv("NTT_PARAMETERS_DIR")`. Have a look at the implementation of `suite.ParametersFile` for example.

Reply:
done